Scripts/parameters/assignment.py

```diff
@@ -186,6 +186,21 @@
         "iht": 1 / 0.497,
     },
 }
+# Factor for converting weekday traffic into yearly day average
+years_average_day_factor = 0.85
+# Factor for converting day traffic into 7:00-22:00 traffic
+share_7_22_of_day = 0.9
+# Noice zone width as function of start noise
+noise_zone_width = [
+    {"threshold": 80, "function": lambda x: 225},
```

Comment:
What do you think about changing `threshold` to explicit `lower` and `upper` boundaries? Instead of `start_noise >= i["threshold"]` in https://github.com/HSLdevcom/helmet-model-system/pull/269/files#diff-2cffa1ae64c4860a3669589e445fcb62e50cda3379c945f94887344fd06f1e12R306 we would have mathematically "noise in [lower, upper)" so something like `noise >= i["lower"] & noise < i["upper"]`. I think it would make debugging easier.

Reply:
Did something like this, hope you like it!